app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,35 @@
+class FoodsController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    @foods = current_user.foods.includes(:user)
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    @food = current_user.foods.build(food_params)
+    if @food.save
+      @food = @food.includes(:user)
+      redirect_to foods_path, notice: 'Food successfully created.'
+    else
```

Comment:
- [Optional] Dear Team, though your food is getting added to the list, I keep getting an error `undefined method `includes' for #<Food .....` error, this is not a professional way of doing things, we need to keep out system error-free and consider the edge cases. Please take a look and make some adjustments. 
![image](https://github.com/torobucci/Recipe_app/assets/69077061/4eec3df8-ed16-4be9-913c-506c9ed7bc17)